python/cudf/cudf/core/column/lists.py

```diff
@@ -176,6 +176,37 @@ def __init__(self, column, parent=None):
             )
         super().__init__(column=column, parent=parent)
 
+    def extract(self, index):
```

Comment:
We should update this to `def get` and follow https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.str.get.html#pandas-series-str-get